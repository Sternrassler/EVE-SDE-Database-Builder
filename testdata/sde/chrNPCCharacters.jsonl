{"characterID":3004,"characterName":"Jamyl Sarum","gender":0,"bloodlineID":4,"description":"Former Empress of the Amarr Empire.","solarSystemID":null,"stationID":null,"corporationID":1000084}
{"characterID":3005,"characterName":"Tibus Heth","gender":1,"bloodlineID":1,"description":"Former leader of the Caldari State.","solarSystemID":null,"stationID":null,"corporationID":1000035}
{"characterID":3006,"characterName":"Sanmatar Shakor","gender":1,"bloodlineID":2,"description":"Leader of the Minmatar Republic.","solarSystemID":null,"stationID":null,"corporationID":1000051}
