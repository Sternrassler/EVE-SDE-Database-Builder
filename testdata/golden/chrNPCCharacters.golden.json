[
  {
    "characterID": 3004,
    "corporationID": 1000084,
    "name": null
  },
  {
    "characterID": 3005,
    "corporationID": 1000035,
    "name": null
  },
  {
    "characterID": 3006,
    "corporationID": 1000051,
    "name": null
  }
]